bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -0,0 +1,324 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "teach/smarton/base.html" %}
+
+{% block page_title %}{{ _('Get smart on cybersecurity') }}{% endblock %}
+{% block page_desc %}{{ _('Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') }}{% endblock %}
```

Comment:
String added 20 October: "Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity."